en_US.json

```diff
@@ -444,6 +446,14 @@
 	"contactListing.column.name": "User Name",
 	"contactListing.column.role": "Role",
 	"contactListing.column.status": "Status",
+	"contactListing.contact.invite.cancel.modal.confirm.text": "Do you want to cancel invite to this user: ",
```

Comment:
I kept the text similar to the suspend and delete confirmation text, which don't have a question mark and use a colon.
The code concatenates the username (if available) or contact id to the text.

I will refactor the relevant texts to use the user place holder and the question mark.

Thanks..

Reply:
Regarding "Do you want to cancel the invitation to <user>"

I agree this sounds better. The reason I framed it as "Do you want to" vs "Are you sure you want to", is because resending or cancelling an invitation isn't a major change unlike activating/suspending/deleting contact.

Note that resending/ cancelling an invite is only applicable to contact with "pending" status with a current invitation.

So unlike other contacts, a user with "pending" status won't have a username. As a result, we will need to use either the contact id or email id in the text message. I felt the contact id is better.

Given the above constraints, what should be the appropriate text?

1) Do you want to cancel the invitation to user 123?
OR
2) Do you want to cancel the invitation to 123 ?

I prefer (1)

Reply:
@chiraags  It would be better to show the email address if that is available.

An ID like "123" won't be very meaningful to a portal user.